packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -341,7 +341,10 @@ export default function PostOptionsMenu({
       ),
       label: hasPostReminder ? 'Edit reminder' : 'Read it later',
       action: () => {
-        openModal({ type: LazyModal.BookmarkReminder, props: { post } });
+        openModal({
+          type: LazyModal.BookmarkReminder,
+          props: { post, feedContextData: { items, queryKey: feedQueryKey } },
```

Comment:
```suggestion
        openModal({
          type: LazyModal.BookmarkReminder,
          props: { post, feedContextData },
```

just so we make sure everything is passed